doc/dev/build_archived_sensfuncs.rst

```diff
@@ -3,26 +3,60 @@
 
 .. _build_archived_sensfuncs:
 
-How to build archived sensitivity functions
-===========================================
+
+Creating archival sensitivity functions
+=======================================
+
+Archival sensitivity functions for instruments can be created by combining 
+multiple sensivity functions from different exposures at different 
+grating tilts to get a wider wavelength coverage. The steps in this
+process are typically:
+
+#. Choose sensitivity functions for grating. The functions should have good wavelength coverage,
+   and low airmass.
+
+#. Start with the bluest sensitivity function and translate the next function up/down to roughly match.
+   One or both function may be truncated to fit.
+
+#. If the gap between sensitivity function isn't smooth (for example on a detector boundary) 
+   a polynomial fit can be used to join them.
+
+There are helper functions in $PYPEIT_DEV/sensfunc_archive/stitcutils.py intended to make this easier. See
+$PYPEIT_DEV/sensfunc_archive/create_deimos_sensfuncs.py $PYPEIT_DEV/sensfunc_archive/stitchdeimos.py for examples of how this was done
+for DEIMOS.
+
+The DEIMOS archived sensitivity functions
+-----------------------------------------
+
+*Disclaimer*:
+The DEIMOS archival sensitivity functions do not provide an absolute flux 
+calibration.  Instead, they are only intended to remove the instrumental 
+response, providing a relative flux calibration up to some unknown 
+normalization.
+
+
+Creating all of the DEIMOS sensitivity functions
+************************************************
 
 1. Make sure you have up to date versions of the `PypeIt
    <https://github.com/pypeit/PypeIt>`_ and `PypeIt Development Suite`_ 
    repositories from GitHub, and that PypeIt :ref:`installing` is complete.
 
 2. Download the reduced slitless source images used to create Keck DEIMOS sensitivity functions from
-   the `PypeIt dev-suite Google Drive`_. The files are located under ``DEIMOS_Dev/Throughput/throughput_gdw/data_products/extract``.
-
-   .. code-block:: bash
+   the `PypeIt dev-suite Google Drive`_. The files are located under ``DEIMOS_Dev/Throughput/throughput_gdw/data_products/``.
 
-        rclone -P copy remote:/PypeIt-development-suite/DEIMOS_Dev/Throughput/throughput_gdw/data_products/extract/ source_files/
-
-3. Run the ``create_deimos_sensfuncs.py`` script.
+3. Run the ``create_deimos_sensfuncs.py`` script. The below examples assumes the files
+   were downloaded to a ``data_products`` directory and will create the archival 
+   sensitivity function files under ``sensfunc_files``.
 
    .. code-block:: bash
 
         mkdir sensfunc_files
-        PypeIt-development-suite/sensfunc_archive/create_deimos_sensfuncs.py all source_files sensfunc_files
+        $PYPEIT_DEV/sensfunc_archive/create_deimos_sensfuncs.py all data_products sensfunc_files
+
+   The above creates spec1d files and sensfunc files from the source images. To re-use already created
```

Comment:
"The above creates PypeIt spec1d files and stitched sensitivity function files (one per each grating) from the source images."  Right?

Reply:
I reworded it to be a bit longer and hopefully more clear:

> The above creates spec1d files and sensfunc files from the reduced slitless source images. It then stitches those sensfunc files together to form sensfuncs that cover a wider wavelength range. This results in one stitched archival sensfunc for each grating (600ZD, 830G, 900ZD, 1200B, and 1200G).